erpnext/stock/doctype/item/item_list.js

```diff
@@ -14,6 +14,31 @@ frappe.listview_settings['Item'] = {
 			return [__("Variant"), "green", "variant_of,=," + doc.variant_of];
 		}
 	},
+	onload: function(me) {
+		me.page.add_action_item(__("Create Website Item(s)"), function() {
```

Comment:
Would this be better? Because there's a button inside Item too named **Publish In Website**. It will be familiar and consistent imo
```suggestion
		me.page.add_action_item(__("Publish in Website"), function() {
```